README.rst

```diff
@@ -3,24 +3,25 @@
 
 Raiden Scenario Player
 ======================
-The Raiden Scneario Player is an integration testing tool written in Python 3. It allows testing of various test scenarios, and is
+The Raiden Scenario Player is an integration testing tool written in Python 3. It allows testing of various test scenarios, and is
 an integral component in the raiden test suite.
 
 Installation
 ------------
 
 Using  ``git`` & ``pip``::
 
-    # Clone the raiden repo and install its development version.
-    git clone http://github.com/raiden-network/raiden
-    cd raiden && make install && cd ..
-    # Clone the scenario player repo, and install on the Raiden dev version.
+    # Clone the scenario-player repository
     git clone http://github.com/raiden-network/scenario-player
-    cd scenario-player
-    pip install ".[dev]"
 
-Usage
------
+    # Install raiden's dev requirements.https://github.com/raiden-network/scenario-player/pull/122
+    pip install -r https://raw.githubusercontent.com/raiden-network/raiden/develop/requirements/requirements-dev.txt
```

Comment:
the requirements-dev.txt is not in the folder itself? Would that make sense? At least in the other Raiden Repos we have that always delivered within

Reply:
The dev requirements arent our own, but raiden's - this way there is less hassle keeping the two in sync.

If `raiden` ever switches to `pyproject.toml` for requirements specifications we could set the scenario-player's `raiden` dependency to `raiden[dev]`, which would then install these requirements automatically, sans the extra step.

Until then I would keep it this way - otherwise it's guaranteed to be out of sync eventually.